src/Components/Title/TitleScreen.js

```diff
@@ -39,21 +40,26 @@ class TitleScreen extends Component {
 
   render () {
     return (
-      <div className="TitleScreen">
-        <div className="LogoWrapper">
-          <div className="Logo FadeInElement"/>
-        </div>
-        <h4>
-          <strong>
-            Ganache
-          </strong>
-          <div className="GanacheVersion">
-            v{this.state.version}
+      <div className="TitleScreenContainer">
+        <div className="TitleScreen">
+          <div className="LogoWrapper">
+            <div className="Logo FadeInElement"/>
           </div>
-        </h4>
+          <h4>
+            <strong>
+              Ganache
+            </strong>
+            <div className="GanacheVersion">
+              v{this.state.version}
+            </div>
+          </h4>
+        </div>
+        <OnlyIf test={this.props.core.systemError != null}>
+          <BugModal systemError={this.props.core.systemError} logs={this.props.logs} />
+        </OnlyIf>
       </div>
```

Comment:
If I don't add the extra div here, the styling from TitleScreen applies.

My thinking was maybe I should clear the styling in BugModal and reapply it so that it's explicit, but that makes maintenance a bit of a drag.